amazon-kinesis/1.0/modules/ROOT/pages/amazon-kinesis-connector-reference.adoc

```diff
@@ -0,0 +1,523 @@
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  The time unit for the *Record TTL* value. |  SECONDS |
+| Thread Pool Size a| Number |  Maximum number of threads with which to configure the native process thread pool. The default value, 0, allows the KPL process to choose the size of the thread pool. |  0 |
+|===
+
+==== Connection Types
+[[Config_Connection]]
+===== Kinesis Connection
+
+Connects to AWS Kinesis through the use of a user-specified access and secret key and, optionally, a role. These credentials securely sign requests to AWS services.
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Proxy Configuration a| <<ProxyConfiguration>> |  Configuration for outbound connections through a proxy. |  |
+| TLS Configuration a| <<Tls>> |  Defines a configuration for TLS, which can be used from both the client and server sides to secure communication for the Mule app. When using the HTTPS protocol, the HTTP communication is secured using TLS or SSL. If HTTPS is configured as the protocol, then, at a minimum, the user must configure the keystore in the `tls:context` child element of the `listener-config`.  |  HTTP|
+| Access Key a| String |  Access key provided by Amazon. |  | x
+| Secret Key a| String |  Secret key provided by Amazon. |  | x
+| Region Endpoint a| String |  Topic region endpoint |  us-east-1 |
+| Role a| <<Role>> |  Role configuration |  |
+| Connection Timeout a| Number |  The amount of time to wait when initially establishing the TCP connection between the connector and Amazon Kinesis server before throwing an exception if the connection fails |  30 |
+| Connection Timeout Time Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  The time unit for the *Connection Timeout* field. |  SECONDS |
+| Response Timeout a| Number |  The maximum time to wait between the time an HTTP request is sent and when a response is received |  30 |
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit for the *Response Timeout* value. |  SECONDS |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  |
+|===
+
+== Supported Operations
+* <<Checkpoint>>
+* <<PutRecord>>
+
+==== Associated Sources
+* <<Listener>>
+
+== Operations
+
+[[Checkpoint]]
+== Checkpoint
+`<kinesis:checkpoint>`
+
+The operation checkpoint position of a listener into the Amazon DynamoDB table. Upon failover, the connector starts fetching records that are located after the check pointed position. The *Checkpoint* does not modify the current position of the listener.
```

Comment:
```suggestion
The operation checkpoint position of a listener into the Amazon DynamoDB table. Upon failover, the connector starts fetching records that are located after the check pointed position. The *Checkpoint*  operation does not modify the current position of the listener.
```
```suggestion
The operation checkpoint position of a listener into the Amazon DynamoDB table. Upon failover, the connector starts fetching records that are located after the check pointed position. The *Checkpoint* does not modify the current position of the listener.
```

Reply:
I'm not sure what the first sentence is trying to say:
`The operation checkpoint position of a listener into the Amazon DynamoDB table`
Does it mean _The *Checkpoint* operation positions a a listener into the Amazon DynamoDB table_? Please confirm with devs.